client/core/trade.go

```diff
@@ -659,10 +675,13 @@ func (t *trackedTrade) redeemMatches(matches []*matchTracker) error {
 			match.id, sellString(t.Trade().Sell), match.Match.Quantity, unbip(t.Prefix().BaseAsset),
 		)
 	}
+
 	// Send the redemption information to the DEX.
+	//coinStrs := make([]string, len(matches))
 	for i, match := range matches {
 		_, _, proof, auth := match.parts()
 		coinID := []byte(coinIDs[i])
+		//coinStrs[i], _ = t.wallets.toWallet.DecodeCoinID(coinIDs[i]) // unless Redeem returns output asset.Coin
```

Comment:
Want to leave this?

Reply:
I have stashed changes.  I'm bouncing between 3 feature branches right now.  :(